build.gradle.kts

```
@@ -65,6 +66,8 @@ dependencies {
     // Mail
     implementation("com.j2html:j2html:$j2htmlVersion")
     implementation("org.springframework.boot:spring-boot-starter-mail")
+    // AWS SNS
+    implementation("software.amazon.awssdk:sns:$awsSNSVersion")
```

`implementation(platform("software.amazon.awssdk:bom:2.24.0"))` 으로 모든 AWS SDK에 대한 버전을 관리하도록 해두어서 따로 버전을 지정하지 않아도 될 것 같아요!
```suggestion
    implementation("software.amazon.awssdk:sns)